src/main/java/org/isf/stat/gui/OperationsListLauncher.java

```diff
@@ -0,0 +1,136 @@
+package org.isf.stat.gui;

```

Comment:
Please add the standard license header to this file; thanks.

Reply:
I cannot see the license header yet, has it been pushed?

Reply:
I can see the header on the remote branch, maybe it's a GitHub bug here